lib/pg_strutil.c

```diff
@@ -382,13 +394,21 @@ GetNextArgument(const char *ptr, char **arg, Oid *argtype, const char **endptr,
 		str = palloc(len + 2);
 		snprintf(str, len + 2, "%c%s", minus ? '-' : '+', startptr);
 
-		/* could be an oversize integer as well as a float ... */
+/* could be an oversize integer as well as a float ... */
+#if PG_VERSION_NUM >= 150000
+		errno = 0;
+		val64 = strtoi64(str, &tmpendptr, 10);
+
+		if (errno == 0 && *tmpendptr == '\0')
+#else
 		if (scanint8(str, true, &val64))
+#endif
 		{
 			/*
-			 * It might actually fit in int32. Probably only INT_MIN can
-			 * occur, but we'll code the test generally just to be sure.
-			 */
+			* It might actually fit in int32. Probably only INT_MIN can
+			* occur, but we'll code the test generally just to be sure.
+			*/
```

Comment:
Need to add a space

Reply:
I made a mistake in the correction. There should be a space in the bottom three lines.